pkg/brctl/brctl.go

```diff
@@ -466,8 +472,9 @@ func Hairpin(bridge string, port string, hairpinmode string) error {
 		hairpinMode = "0"
 	}
 
-	if err := setPortBrportValue(port, BRCTL_HAIRPIN, []byte(hairpinMode)); err != nil {
-		return fmt.Errorf("setPortBrportValue: %w", err)
+	err := setPortValue(port, BRCTL_HAIRPIN, []byte(hairpinMode))
+	if err != nil {
+		return fmt.Errorf("set hairpin mode: %w", err)
```

Comment:
Could you tell me what you expanded the error handling here?

Reply:
To be consistent with the rest of the code. Further, that version generally serves better readability on long expressions.